model/move_embedding.py

```diff
@@ -10,7 +10,7 @@
 # Note that there are 901 total moves in MOVE_NAME_TO_NUM dictionary. However, the identifers 'num' do NOT
 # range from 0 to 900 as some are missing. Therefore, vocab_size will be slightly higher than actual number
 # of possible moves to allow lookup without out of index error in the Embedding.
-VOCAB_SIZE = max(MOVE_NAME_TO_NUM.values())
+VOCAB_SIZE = max(MOVE_NAME_TO_NUM.values()) + 1
```

Comment:
why do we need the +1 here? Isn't the emptymove added directly to `moves.csv`?

Reply:
actually never mind, max value could still be OOB if vocab size is max value.

Reply:
I believe we would need similar +1 in ability embeddings too then

Reply:
yes very true, slipped my mind